src/ewatercycle/config/__init__.py

```diff
@@ -81,6 +91,6 @@
     # apptainer pull docker://ewatercycle/wflow-grpc4bmi:2020.1.1
 """
 
-from ._config_object import CFG, DEFAULT_CONFIG, SYSTEM_CONFIG, USER_HOME_CONFIG, Config
+from ._config_object import CFG, SYSTEM_CONFIG, USER_HOME_CONFIG, Configuration
 
-__all__ = ["CFG", "Config", "DEFAULT_CONFIG", "SYSTEM_CONFIG", "USER_HOME_CONFIG"]
+__all__ = ["CFG", "Configuration", "SYSTEM_CONFIG", "USER_HOME_CONFIG"]
```

Comment:
Would it make sense to move documentation to the user guide, and reduce the config to a single-file module called `config.py`?

Reply:
I moved the content of src/ewatercycle/config/_config_object.py to src/ewatercycle/config/__init__.py in c86f3b75a58566167d74b416d47e37414924bd63.  
I also added a link from docs/system_setup.rst to api docs.
To not increase the public API I prefixed private things with underscore.

Moving it to src/ewatercycle/config.py could be done once src/ewatercycle/config/_lisflood_versions.py has been moved to its own plugin/repo.